Orange/widgets/data/owfile.py

```diff
@@ -257,29 +257,36 @@ def browse_file(self, in_demos=False):
 
     # Open a file, create data from it and send it over the data channel
     def load_data(self):
+        # We need to catch any exception type since anything can happen in
+        # file readers
+        # pylint: disable=broad-except
         self.editor_model.set_domain(None)
-        self.reader = self._get_reader()
-        if not self.reader:
-            self.data = None
```

Comment:
`self.data` used to be cleared in case of errors, now it is not anymore...

This is used in reports for example (maybe elsewhere too), which now report on previous data, even after trying to open something else and failing.